### ISSUE TITLE
17:17: error: oui.h: No such file or directory  #include "oui.h"                  ^

### ISSUE THREAD

**Issue by tytower** [spacehuhn/esp8266_deauther#1139]
>  ```Deauth:17:17: error: oui.h: No such file or directory
 #include "oui.h"
                 ^
compilation terminated.
exit status 1
oui.h: No such file or directory```

<details>
  <summary> Compile log (...)</summary>

  

</details>
<br><br><br>

**Comment by tytower** [comment]
This is still happening on an arduino compile following the instructions to the letter. The advice to download as a tar file is poppycock

**Comment by tytower** [comment]
Don't be smart . RTFM Ive read all the install notes . If this comes up regularly then be decent enough to explain yourself or sticky the thing like the ArduinoJson massage has been done.

**Comment by tytower** [comment]
OK so the guy likes to be a smart A . 
Now the answer to this for any who come after me looking for "oui.h" is that it IS in the downloaded zip file.
My mistake was to save the sketch afterwards by copying the ".ino" text into a new sketch and saving it and then trying to compile it.

It could not find the deauth files that way . 
I found it in the sketchbook in its original downloaded form and when I started the sketch from there it compiled fine .

So that error just tells you its not finding the files that came with the sketch.
